queue_services/entity-emailer/src/entity_emailer/email_templates/common/nr-footer-html.html

```diff
@@ -0,0 +1,10 @@
+[[divider.html]]
+<p style="margin-top: 25px !important;"></p>
+<footer>
+  <div class="bold">Business Registry</div>
+  <div>BC Registries and Online Services</div>
+  [[20px.html]]
+  <div>Toll Free: <a href="tel:1-877-370-1033">1-877-370-1033</a></div>
+  <div>Victoria Office: <a href="tel:250-370-1033">250-370-1033</a></div>
+  <div>Email: <a href="mailto:BCRegistries@gov.bc.ca">BCRegistries@gov.bc.ca</a></div>
+</footer>
```

Comment:
This filename is confusing. I recommend a name that indicates when this file should be used instead of `nr-footer.html`. (And/or rename the other file accordingly.)

Reply:
What do you recommend? ;)

Reply:
Well, when should you use one footer versus the other?

It looks to me like one is actual HTML but the other is a template.

Reply:
maybe just leave it. We will convert this one later once I finish the new version of notify service.

Reply:
Sure.